abstract_importer.gemspec

```diff
@@ -17,8 +17,8 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "activerecord", ">= 5.0"
-  spec.add_dependency "activesupport", ">= 5.0"
+  spec.add_dependency "activerecord", "< 6.0"
+  spec.add_dependency "activesupport", "< 6.0"
```

Comment:
Did we figure out that `activerecord-insert_many` depends on `activerecord >= 5.0`?